docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -109,3 +109,23 @@ NOTE: You can also use **kubectl edit** to add the build-nudges-ref instead of p
 
 * Ensure that the existing references in the repositories of your components are correct.
 * Try running the necessary commands again, and ensure you have the correct syntax.
+
+== Using custom renovate config
```

Comment:
It's really not clear how renovate is related to nudges. We should explain it here.

Reply:
ok